packages/metro-react-native-babel-preset/src/configs/main.js

```diff
@@ -164,11 +164,15 @@ const getPreset = (src, options) => {
   }
 
   if (!options || options.enableBabelRuntime !== false) {
+    // Allows configuring a specific runtime version to optimize output
+    const isVersion = typeof options?.enableBabelRuntime === 'string';
+
     extraPlugins.push([
       require('@babel/plugin-transform-runtime'),
       {
         helpers: true,
         regenerator: !isHermes,
+        version: isVersion ? options.enableBabelRuntime : '7.0.0-beta.0',
```

Comment:
Can we avoid baking in the plugin's default fallback version?
```suggestion
        ...(isVersion && {version: options.enableBabelRuntime}),
```

Reply:
Yes definitely - I like this mechanism for initializing a specific property only under certain circumstances.